### PR TITLE
feat: Finalise loaders

### PR DESCRIPTION
**Pull request by henriknorin-ftrack**
Resolves : 

* CLICKUP-3xr1qy5

This PR has been tested on :

- [X] Windows.
- [ ] MacOs.
- [ ] Linux.

## Changes

- Finalised and consolidated geometry, rig and animation loaders
- Have rename of imported asset as an option.

## Test

_Note: needs to be tested with the corresponding branch on definition and unreal._

- Export geometry, rig and animation from other DCC, FBX and Alembic.
- Test loading and validating all combinations in Unreal.

### CODE REVIEW

File: resource/plugins/unreal/python/loader/importers/unreal_fbx_rig_loader_importer.py [hunk 1]
```diff
@@ -93,19 +66,50 @@ def run(self, context_data=None, data=None, options=None):
...
                 )
 
-        task.replace_existing = options.get('ReplaceExisting', True)
-        task.automated = options.get('Automated', True)
-        task.save = options.get('Save', True)
-
         import_result = unreal_utils.import_file(task)
         self.logger.info('Imported FBX rig: {}'.format(import_result))
-        loaded_mesh = unreal.EditorAssetLibrary.load_asset(import_result)
+        loaded_skeletal_mesh = unreal.EditorAssetLibrary.load_asset(
```
Comment (lluisCM) [inline resource/plugins/unreal/python/loader/importers/unreal_fbx_rig_loader_importer.py:71]: I don't think this is needed. Please double check. 

I think you can use asset.get_path_name()  and asset.skeleton and asset.physics_asset without having to load the asset. I assume that import_result is an asset type right?

File: resource/plugins/unreal/python/loader/importers/unreal_abc_rig_loader_importer.py [hunk 2]
```diff
@@ -73,10 +49,6 @@ def run(self, context_data=None, data=None, options=None):
                     'skeleton', skeletonAD.get_asset()
                 )
 
-        task.replace_existing = options.get('ReplaceExisting', True)
-        task.automated = options.get('Automated', True)
-        task.save = options.get('Save', True)
-
         import_result = unreal_utils.import_file(task)
         self.logger.info('Imported Alembic rig: {}'.format(import_result))
         loaded_skeletal_mesh = unreal.EditorAssetLibrary.load_asset(
```
Comment (lluisCM) [inline resource/plugins/unreal/python/loader/importers/unreal_abc_rig_loader_importer.py:54]: I think you don't need to load the asset, please read my other comment.

**Review by lluisCM — APPROVED** [review]
LGTM, 

But still think that there is no need to load the asset. the load asset method loads the asset in cache, but you have other methods to get the asset object which is what you need in your code.

File: resource/plugins/unreal/python/loader/importers/unreal_fbx_rig_loader_importer.py (hunk 1 above)
Comment (henriknorin-ftrack) [reply resource/plugins/unreal/python/loader/importers/unreal_fbx_rig_loader_importer.py:71]: I need it to load the asset into an object so I can retrieve the associated objects (skeleton and physics asset in this case)

File: resource/plugins/unreal/python/loader/importers/unreal_abc_rig_loader_importer.py (hunk 2 above)
Comment (henriknorin-ftrack) [reply resource/plugins/unreal/python/loader/importers/unreal_abc_rig_loader_importer.py:54]: See my comment above

**Pull request merged**